tm2/pkg/crypto/keys/client/broadcast.go

```diff
@@ -87,6 +89,7 @@ func execBroadcast(cfg *BroadcastCfg, args []string, io commands.IO) error {
 		io.Println("GAS USED:  ", res.DeliverTx.GasUsed)
 		io.Println("HEIGHT:    ", res.Height)
 		io.Println("EVENTS:    ", string(res.DeliverTx.EncodeEvents()))
+		io.Println("TX HASH:   ", url.QueryEscape(base64.StdEncoding.EncodeToString(res.Hash)))
```

Comment:
@zivkovicmilos During testing, I observed that when the hash returned in base64 format contains special characters like this one, `8y9J740MXZUhlgUv+Kxt+yVRqtTuFjHpNdLZPkZVhR4=`, the hash parsed from the HTTP server side converts special characters into spaces. This function will help transform special characters into their corresponding base64 string `8y9J740MXZUhlgUv%2BKxt%2ByVRqtTuFjHpNdLZPkZVhR4%3D` while preserving the integrity of the original string